### ISSUE TITLE
Show breaking news on any page (not only the dashboard)

### ISSUE THREAD

**Issue by funkycram** [humhub-contrib/breakingnews#24]
Some Humhub instance don't use the dashboard (the home page is changed in the settings).
For them, the breaking news never displays.

I suggest to change the event in `config.php` to:
```php
        [
            'class' => TopMenuRightStack::class,
            'event' => TopMenuRightStack::EVENT_BEFORE_RUN,
            'callback' => [Events::class, 'onTopMenuRightStackBeforeRun']
        ],
```

If you want, I can send a PR.
If you want, I can take advantage of this PR to move the event-related methods from Module.php to Events.php.

**Comment by luke-** [comment]
@funkycram  What do you think about integrating it into `humhub\widgets\LayoutAddons`? This should be available on every page.

**Comment by luke-** [comment]
A PR would be great here, of course! :-)

**Comment by luke-** [comment]
Released in 1.0.7
